### ISSUE TITLE
Add overlay?

### ISSUE THREAD

**Issue by EmilyLars** [miykael/gif_your_nifti#8]
Might it be possible to visualize two images, anatomical base with an overlay with a different color scheme?

**Comment by ofgulban** [comment]
Hi @EmilyLars , thanks for opening this issue. Do you mean an overlay to show activation maps etc. ? This is possible but needs to be implemented. I might have a look into it.

**Comment by EmilyLars** [comment]
Yes - I do structural imaging but same idea - I'm imagining a figure where I've color coded certain brain structures or cluster to display my results. Could be really useful to use your tool to scan through instead of having to show all the different slices. Thank you!

**Comment by miykael** [comment]
Yes, that might be useful. However, the problem with these gifs are that you cannot really stop them. So for some results the loop might be too fast or not focused enough.

Perhaps something like [nilearn's plotting](https://nilearn.github.io/plotting/index.html) rountines might be better for this? For example the [glass_brain plot](https://nilearn.github.io/auto_examples/01_plotting/plot_demo_glass_brain_extensive.html) could be used for a quick overview of your results. Nilearn also has an [interactive visualization plot feature](https://nilearn.github.io/plotting/index.html#interactive-visualization-of-statistical-map-slices) which allows you to scroll through the images. And if you want to have everything automaed, what about [atlasreader](https://github.com/miykael/atlasreader) to quickly create a multitude of figures and output tables?

**Comment by ofgulban** [comment]
@miykael I think @EmilyLars means the gif as a complementary figure (to other interactive visualizers) to give a quick impression of the activation blobs. I can imagine that e.g. in a blog post such animation would convey the 3D extend of the more easily.

**Comment by EmilyLars** [comment]
Yes I agree - alone it would be hard to appreciate the results but could be a useful complement for talks for example

**Comment by araikes** [comment]
@miykael, 

I'll just give this a small bump. I working on mask generation for rodent brains and the GIFs work with my brains. Being able to overlay the mask or an atlas would be helpful as a QC step. If there's a way to that, I'd be very interested.
